WISHLIST.md

```diff
@@ -0,0 +1,64 @@
+WISHLIST to make LUYA a better CMS
+==============
+
+Simply put your suggestions and wishes for improvements in this basic text file to help us to make LUYA your favorite futur CMS.
+
+
+Installation
+-------------------
+
+- integrate composer fxp/composer-asset-plugin installation and availibilty test in composer create project.
+
+
+
+Console
+-------------------
+- beautify console appearance and shorten luya console commands (tab complete doesn't work OSX)
+- numeric select option for selections
+
+
+
+> Questions / Ideas
+
+Is it the best choice to use yii2 console? <br/>
+How 'hackable' ist the yii2 console application?
+
+> Examples
+
+[Simple python cli tool ideas](https://kushaldas.in/posts/building-command-line-tools-in-python-with-click.html)
+
+Blocks
+-------------------
+
+- auto generation for fields from console cms/block/create
+- auto generation for fields in view file from console cms/block/create
+- improve password entering during install (visibility and validation)
+- implement and show unique block indentifier
+- make block title content contexual for better recognition and better block organisation ( as well in right sidebar, f.e. copy)
+
+
+Administration
+-------------------
+
+- redesign and restructure
```

Comment:
you should maybe specify more exactly what you want to do. its like telling a car company to build a new car without saying what you want.